TheBTeam.ConsoleApp/TransactionViewer.cs

```diff
@@ -9,24 +9,30 @@ public static class TransactionViewer
     {
         public static void ViewTransaction(List<Transaction> transactions)
         {
-            var textPaddingWidth = 23;
+            var textPaddingWidth = 20;
             var paddingChar = ' ';
+            var numberOfCollumn = 7;
+            Console.WriteLine(("").PadRight(textPaddingWidth * numberOfCollumn, '='));
             Console.WriteLine($"|{"FirstName".PadRight(textPaddingWidth, paddingChar)} " +
-                                  $"|{"LastName".PadRight(textPaddingWidth, paddingChar)} " +
-                                  $"|{"Type".PadRight(textPaddingWidth, paddingChar)}" +
-                                  $"|{"Category".PadRight(textPaddingWidth, paddingChar)}" +
-                                  $"|{"Currency".PadRight(textPaddingWidth, paddingChar)}" +
-                                  $"|{"Amount".PadRight(textPaddingWidth, paddingChar)}");
-            Console.WriteLine();
+                              $"|{"LastName".PadRight(textPaddingWidth, paddingChar)} " +
+                              $"|{"Type".PadRight(textPaddingWidth, paddingChar)}" +
+                              $"|{"Category".PadRight(textPaddingWidth, paddingChar)}" +
+                              $"|{"Currency".PadRight(textPaddingWidth, paddingChar)}" +
+                              $"|{"Balance".PadRight(textPaddingWidth, paddingChar)}" +
+                              $"|{"Amount".PadRight(textPaddingWidth, paddingChar)}");
+            Console.WriteLine(("").PadRight(textPaddingWidth * numberOfCollumn, '='));
             foreach (var item in transactions)
             {
                 Console.WriteLine($"|{item.User.FirstName.ToString().PadRight(textPaddingWidth, paddingChar)} " +
                                   $"|{item.User.LastName.ToString().PadRight(textPaddingWidth, paddingChar)} " +
                                   $"|{item.Type.ToString().PadRight(textPaddingWidth, paddingChar)}" +
                                   $"|{item.Category.ToString().PadRight(textPaddingWidth, paddingChar)}" +
                                   $"|{item.Currency.ToString().PadRight(textPaddingWidth, paddingChar)}" +
+                                  $"|{item.User.Balance.ToString().PadRight(textPaddingWidth, paddingChar)}" +
                                   $"|{item.Amount.ToString().PadRight(textPaddingWidth, paddingChar)}");
             }
+            Console.WriteLine(("").PadRight(textPaddingWidth * numberOfCollumn, '='));
+            Console.WriteLine($"Press any key to continue");
```

Comment:
here you can add readkey

Reply:
Herę readkey is not needed , is in main menu